src/manifolds/ProbabilitySimplex.jl

```diff
@@ -354,6 +354,45 @@ function parallel_transport_to!(M::ProbabilitySimplex{N}, Y, p, X, q) where {N}
     return amplitude_to_simplex_diff!(M, Y, q_s, Ys)
 end
 
+@doc raw"""
+    rand(::ProbabilitySimplex; vector_at=nothing, σ::Real=1.0)
+
+
+When `vector_at` is `nothing`, return a random (uniform over the Fisher-Rao metric)
```

Comment:
I'm not certain what the right terminology here is, but I don't think this is it. Maybe something like
```suggestion
When `vector_at` is `nothing`, return a random (uniform over the Fisher-Rao metric; that is, uniform with respect to the `n`-sphere whose positive orthant is mapped to the simplex).
```
If the bijection between the positive orthant and the simplex is somewhere documented already, we could link to that docstring.

But I'll do a literature search to see if there's a more specific name we can give here.

Reply:
I think that can be done in a follow-up PR? This one is already taking way too much time.